README.md

```diff
@@ -43,9 +43,12 @@ manually, try the [manual build setup][manual-build].
 
  - Download and run [`rustup-init.exe`](https://win.rustup.rs/) then follow the onscreen instructions.
  - Install [chocolatey](https://chocolatey.org/)
- - Run `choco install support\windows\chocolatey.config`
-   *This will install CMake, Git, LLVM, Ninja, NuGet, Python and the Visual Studio 2019 Build Tools.*
- - Run `mach bootstrap-gstreamer`
+ - Run `mach bootstrap`
+  - *This will install CMake, Git, LLVM, Ninja, NuGet, Python and the Visual Studio 2019 Build Tools
```

Comment:
Will LLVM be installed via choco as this comment suggests? I see we have removed it in choco.config

Reply:
Oh, nice catch. I've fixed this now.